person.rb

```diff
@@ -3,17 +3,21 @@
 require_relative './nameable'
 require_relative './trimmer_Decorator'
 require_relative './capitalize_Decorator'
+require_relative './rental'
+require_relative './book'
+require_relative './classroom'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :borrowings
```

Comment:
- [ ] Setters for `@borrowings` should be present in the Person class since they are accessed in the Rental class.